### ISSUE TITLE
requestIP return null when using sign cookie

### ISSUE THREAD

**Issue by armada45-pixel** [elysiajs/elysia#655]
### What version of Elysia.JS is running?

1.0.21

### What platform is your computer?

`Alpine Linux v3.18` on `Docker 4.30.0 ( Desktop )` on `WSL version: 2.1.5.0` on `Microsoft Windows NT 10.0.22631.0 x64`

### What steps can reproduce the bug?

when have cookie need to sign, `socketAddress` from `requestIP` will return will

### What is the expected behavior?

requestIP return normal socketAddress object

### What do you see instead?

`null`

### Additional information

```ts
import { Elysia } from "elysia"

const app = new Elysia({
  name: "mainApp",
  serve: {
    hostname: "0.0.0.0",
    port: 8082,
  },
  cookie: {
    sign: true, // If false requestIP return object of socketAddress
    secrets: process.env.COOKIE_KEY,
  },
})
  .use((app) =>
    app.derive(({ request }) => ({
      ip: app.server?.requestIP(request)?.address,
    })),
  )
  .derive({ as: "global" }, ({ cookie: { someMilk } }) => ({
    get milkType() {
      const value = "chocolate milk"
      someMilk.set({
        value: value,
      })
      return value
    },
  }))
  .get("/", ({ ip }) => {
    return ip + "some message"
  })
  .listen({}, ({ development, hostname, port }) => {
    console.log(
      `🦊 Elysia is running at ${hostname}:${port} ${
        development ? "🚧 in development mode!🚧" : ""
      }`,
    )
  })
  ```

**Comment by SaltyAom** [comment]
Blocking on https://github.com/oven-sh/bun/issues/6613

**Comment by kravetsone** [comment]
[Fixed by Bun@1.1.27](https://bun.sh/blog/bun-v1.1.27#fixed-server-requestip-after-await)

```ts
import { Elysia } from "elysia";

const app = new Elysia({
    name: "mainApp",
    serve: {
        port: 8082,
    },
    cookie: {
        sign: true, // If false requestIP return object of socketAddress
        secrets: process.env.COOKIE_KEY || "ok",
    },
})
    .use((app) =>
        app.derive(({ request }) => ({
            ip: app.server?.requestIP(request)?.address,
        }))
    )
    .derive({ as: "global" }, ({ cookie: { someMilk } }) => ({
        get milkType() {
            const value = "chocolate milk";
            someMilk.set({
                value: value,
            });
            return value;
        },
    }))
    .get("/", ({ ip }) => {
        return ip + "some message";
    })
    .listen({}, ({ development, hostname, port }) => {
        console.log(
            `🦊 Elysia is running at ${hostname}:${port} ${
                development ? "🚧 in development mode!🚧" : ""
            }`
        );
    });

```
![image](https://github.com/user-attachments/assets/d52f69b5-ea50-400f-bc08-a090a9721144)

**Comment by armada45-pixel** [comment]
Wait for testing.
